gatsby-config.js

```diff
@@ -49,9 +49,23 @@ module.exports = {
     `gatsby-transformer-sharp`,
     `gatsby-plugin-sharp`,
     {
-      resolve: `gatsby-plugin-google-analytics`,
+      // How to configure? https://www.gatsbyjs.com/plugins/gatsby-plugin-google-gtag/
+      // Video: https://www.youtube.com/watch?v=Dwi99jtl3Fs
+      resolve: 'gatsby-plugin-google-gtag',
```

Comment:
There are missing package.json and package-lock files in this PR.
This is the reason why this check is not passing
 https://github.com/flotiq/gatsby-starter-wordpress-blog/pull/6/checks?check_run_id=6503190669

@veto221 can you add missing files?